test/e2e/utils/tg.ts

```diff
@@ -1,12 +1,13 @@
 import { fetch } from 'undici';
 
 // tgpak_${base32([id]_[testX-...])}
+// They're split so they won't be picked by an automated code scanner
```

Comment:
And it uses Tolgee production while testing?

Reply:
No, it uses a local test container spin up by `scripts/startDocker.mjs` and configured with `scripts/tolgee.env`

Reply:
The fact I split them like that is to prevent false positives more than to silence a real potential warning; these keys should never be valid on a real-world Tolgee instance (which is why it's safe to have them)

Reply:
Ah, ok. Then it's fine. I was a bit confused 😄